src/components/ServiceStandard.tsx

```diff
@@ -0,0 +1,143 @@
+import React from "react";
+import { ServiceCardType } from "@components/ServiceCard";
+import styled from "styled-components";
+import Prosthetics from "@assets/icons/home/prosthetics.png";
+import Enzymes from "@assets/icons/home/enzyme.png";
```

Comment:
please download and use icons as svgs, they were png based on the former design

Reply:
Done